src/main/resources/scripts/homes.sk

```diff
@@ -3,20 +3,38 @@
 # define a home location with /sethome and teleport to that location whenever they want to with /home.
 #
 
-command /sethome:
-	description: Set your home
+command /sethome <text>:
+	description: Sets a home to your location with the name you specify.
 	permission: skript.home
 	executable by: players
 	trigger:
-		set {homes.%player%.location} to location of player
-		message "Set your home to <grey>%location of player%<reset>"
+		if {homes.%player%::%arg-1%} is set:
+			send "A home with that name is already set, please specify a new name."
+		else:
+			set {homes.%player%::%arg-1%} to location of player
+			message "Set your home '%arg-1%' to %location of player%"
 
-command /home:
-	description: Teleport yourself to your home
+command /deletehome <text>:
+	description: Deletes the home you specify if it exists.
 	permission: skript.home
+	aliases: delhome
 	executable by: players
 	trigger:
-		if {homes.%player%.location} is not set:
-			message "You have not set your home yet!"
-			stop trigger
-		teleport player to {homes.%player%.location}
\ No newline at end of file
+		if {homes.%player%::%arg-1%} is set:
+			delete {homes.%player%::%arg-1%}
+			send "Successfully deleted your home '%arg-1%'."
+		else:
+			send "You don't have a home with that name."
+
+command /home <text>:
+	description: Teleport yourself to your home.
+	permission: skript.home
+	executable by: players
+	trigger:
+		if {homes.%player%::%arg-1%} is set:

```

Comment:
`{homes::%player%::%arg-1%}` would be preferred even if it the same thing.

Reply:
Will convert to uuid and new variable.